applications/DEMApplication/tests/test_DEM_schemes.py

```diff
@@ -38,7 +38,7 @@ def GetProblemNameWithPath(self):
         return os.path.join(self.main_path, self.DEM_parameters["problem_name"].GetString())
 
     def CheckValues(self, x_vel, dem_pressure):
-        tol = 1.0e-18
+        tol = 1.0e-10
```

Comment:
increasing tolerance. Why? If it was passing the tests before, the tolerance seems ok, no?